src/_layouts/indepth_guide.erb

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+---
+
+<% @entries = site.data.guides_sidebar[resource.data.sidebar_name] %>
+<%= render Shared::Sidebar.new(data: @entries, current: resource) do |sidebar| %>
+  <% sidebar.slot :before do %>
+    <h1>
+      <%= resource.data.sidebar_title %>
+      <span class="version"><%= resource.data.sidebar_version %></span>
+    </h1>
+  <% end %>
+  <% sidebar.slot :after do %>
+    <%= render Shared::BumpCta.new(slug: site.data.slug) %>
+  <% end %>
+<% end %>
+<main class="guide-content wrapper" data-wrapper-size="small">
+  <button data-button-style="secondary" data-button-size="small" data-toggle-target="button" data-action="toggle#click">
```

Comment:
You don't need to specify the controller for the `data-action`?

Reply:
The controller is `toggle`, I vonluntarily kept a very generic name because it just toggles any stuff given a button and a target.